apps/central-scan/backend/src/fujitsu_scanner.test.ts

```diff
@@ -116,19 +116,20 @@ test('fujitsu scanner can scan with letter size', () => {
       '--page-width',
       '215.872',
       '--page-height',
-      '279.364',
+      '336.506',
+      '--bgcolor=black',
     ])
   );
 });
 
-test('fujitsu scanner can scan with legal size', () => {
+test('fujitsu scanner can scan with expected params on legal size election', () => {
```

Comment:
For consistency with letter above.
```suggestion
test('fujitsu scanner scans with expected params on legal-size election', () => {
```